### PR TITLE
Three Receivers Calibration: clarify SOLT procedures and source port.

### PR DESCRIPTION
**Pull request by biergaizi**
This update clarifies the Correct SOLT Calibration Procedure for 1.5-port VNAs. First, I added a reference to the SOLT example. I also explained the difference between a standard SOLT calibration and a three-receiver calibration, including the fact that SOL at port 2 is not needed (unless there's need to calibrate isolation), and that the results are still saved as 2-port network, but S22 and S12 will contain no meaningful data.

This PR clarifies the question reported in Issue #629.

### CODE REVIEW

**Comment by biergaizi** [comment]
Just pushed another commit on `source_port`

Pointing out the fact that TwoPortOnePath assumes port 1 is the active port and S11/S21 is the valid data, but this assumption can be changed to S12/S22 by setting "source_port=2".

**Review by Ttl — APPROVED** [review]

**Review by jhillairet — APPROVED** [review]

File: doc/source/examples/metrology/Calibration With Three Receivers.ipynb [hunk 1]
```diff
@@ -70,7 +70,20 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Measure the Calibration Standands and DUT\n",
+    "\n",
+    "First, one measures the S-parameters of the Open, Short, and Load standards on the VNA according to the standard SOLT calibration procedures. We assume the reader is familiar with this task in scikit-rf. If not, it's fully described in the [SOLT example](./SOLT.ipynb).\n",
+    "\n",
+    "When calibrating a three-receiver VNA, the process is almost identical. But note that there are a few minor differences. First, the Open, Short, and Load standards are only measured at port 1, not port 2. The VNA cannot measure in the reverse orientation, so there's no reason to perform OSL calibration at port 2 (unless one needs the optional isolation calibration). Nevetheless, scikit-rf still expects two-port networks as input, we therefore still save all results as two-port networks - only $S_{11}$ and $S_{21}$ are the actual measurements - $S_{12}$ and $S_{22}$ contain no meaningful data, but it's okay.\n",
```
Comment (Vinc0110) [inline doc/source/examples/metrology/Calibration With Three Receivers.ipynb:84]: To clarify even more: S12 and S22 will be overridden in the `TwoPortOnePath` constructor to make the incomplete 2-port compatible for the `TwelveTerm` routine (by making it symmetrical with S22=S11, S12=S21). https://github.com/scikit-rf/scikit-rf/blob/7ad561755afbb6c939baa4c68a0d49b96d4c5684/skrf/calibration/calibration.py#L1825-L1829

So, the user can put any data they want for S12 and S22, most people will probably chose S12=S22=0, or they can directly put S12=S21, S22=S11. It does not matter.

**Review by Vinc0110 — APPROVED** [review]
Great. I only have one minor suggestion.

File: doc/source/examples/metrology/Calibration With Three Receivers.ipynb (hunk 1 above)
Comment (biergaizi) [reply doc/source/examples/metrology/Calibration With Three Receivers.ipynb]: I just changed the wording to:

> we therefore still save all results as two-port networks. In this case, only $S_{11}$ and $S_{21}$ are the actual measurements, $S_{12}$ and $S_{22}$ are unused, not meaningful, and can contain arbitrary data. One can use all-zero values as placeholders. Internally, scikit-rf will set $S_{22} = S_{11}$ and $S_{12} = S_{21}$ for the purpose of calculations.

**Comment by jhillairet** [comment]
@biergaizi is it OK for merging?

**Comment by biergaizi** [comment]
Yes.

**Pull request merged**